addons/common/functions/fnc_getCursorPosition.sqf

```diff
@@ -0,0 +1,38 @@
+#include "script_component.hpp"
```

Comment:
This function is very similar to `getPosFromScreen`. We should just have one that satisfies all of the requirements.

Reply:
https://github.com/zen-mod/ZEN/pull/496#discussion_r558844225